css-positioning/absolute.css

```diff
@@ -0,0 +1,37 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  right: 10px;
+  top: 10px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: relative;
+  top: 200px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+  position: absolute;
+  width: 300px;
+  height: 300px;
```

Comment:
instead of `width` and `height` you should use a combination of `top/right/bottom/left`

What happens if you set all of those values to `0`? You'll basically be telling the `.overlay` to take up the entire size of it's parent container!

Reply:
👍